### PR TITLE
Missing features

### PR DESCRIPTION
**Pull request by bysiakhubert**
- remove run button
- page with participants

### CODE REVIEW

File: src/main/java/com/virtuslab/workshops/kotlin/run/RunService.java [hunk 1]
```diff
@@ -56,6 +58,7 @@ public RunDetails participateInRun(Integer runId) {
                 .orElseThrow(() -> new IllegalStateException("Couldn't participate in this run"));
     }
 
+    //    TODO [hbysiak] we shouldn't return entity
```
Comment (gregorius2514) [inline src/main/java/com/virtuslab/workshops/kotlin/run/RunService.java:61]: I think at the moment we have mapper for `Run` -> `RunDetails`. We could use it here with minimum of work effort.
What do you think about it?

File: src/main/java/com/virtuslab/workshops/kotlin/run/RunService.java [hunk 2]
```diff
@@ -77,6 +80,20 @@ public RunDetails findById(Integer id) {
                 .orElseThrow(() -> new IllegalStateException("Couldn't find run"));
     }
 
+    public Pair<String, List<SkinnyUserDto>> getRunNameAndParticipants(Integer id) {
+        return runRepository.findById(id)
+                .map(run -> Pair.of(
+                        run.getName(),
+                        run.getParticipants().stream()
+                                .map(user -> new SkinnyUserDto(
```
Comment (gregorius2514) [inline src/main/java/com/virtuslab/workshops/kotlin/run/RunService.java:88]: #whatever 
I would extract it to class or method.

File: src/main/java/com/virtuslab/workshops/kotlin/run/RunService.java [hunk 3]
```diff
@@ -77,6 +80,20 @@ public RunDetails findById(Integer id) {
...
+                                        user.getFirstName(),
+                                        user.getLastName(),
+                                        user.getEmail()
+                                ))
+                                .collect(Collectors.toList())))
```
Comment (gregorius2514) [inline src/main/java/com/virtuslab/workshops/kotlin/run/RunService.java:93]: #whatever 
`toList` could we use static import here?

File: src/main/java/com/virtuslab/workshops/kotlin/run/Run.java [hunk 4]
```diff
@@ -40,7 +40,7 @@
     private LocalTime startTime;
 
     @ManyToMany(mappedBy = "runsInWhichParticipates",
-            cascade = {CascadeType.MERGE, CascadeType.PERSIST})
+            cascade = {CascadeType.MERGE, CascadeType.PERSIST, CascadeType.REMOVE})
```
[the remark this answers is not in the document]
Comment (bysiakhubert) [reply src/main/java/com/virtuslab/workshops/kotlin/run/Run.java:43]: done

File: src/main/java/com/virtuslab/workshops/kotlin/run/RunService.java (hunk 1 above)
Comment (bysiakhubert) [reply src/main/java/com/virtuslab/workshops/kotlin/run/RunService.java:61]: Good idea, done.

File: src/main/java/com/virtuslab/workshops/kotlin/run/RunService.java (hunk 2 above)
Comment (bysiakhubert) [reply src/main/java/com/virtuslab/workshops/kotlin/run/RunService.java:88]: It leaves it as it is for now.

File: src/main/java/com/virtuslab/workshops/kotlin/run/RunService.java (hunk 3 above)
Comment (bysiakhubert) [reply src/main/java/com/virtuslab/workshops/kotlin/run/RunService.java:93]: done

**Pull request merged**